Wabbajack.App.Wpf/View Models/Installers/InstallerVM.cs

```diff
@@ -155,7 +156,26 @@ public class InstallerVM : BackNavigatingVM, IBackNavigatingVM, ICpuStatusVM
     public ReactiveCommand<Unit, Unit> BeginCommand { get; }
     
     public ReactiveCommand<Unit, Unit> VerifyCommand { get; }
-    
+
+
+    private bool _isKeyPressed;
+    public bool IsKeyPressed
+    {
+        get => _isKeyPressed;
+        set => this.RaiseAndSetIfChanged(ref _isKeyPressed, value);
+    }
+
+    private bool _UnrecognisedFilesPresent;
+    public bool UnrecognisedFilesPresent
```

Comment:
Instead of creating private / public properties with RaiseAndSetIfChanged, you should use the [Reactive] property attribute.